internal/local/activate_test.go

```diff
@@ -68,7 +68,7 @@ func TestActivation(t *testing.T) {
 
 func TestActivateCCM(t *testing.T) {
 	f := &flags.Flags{}
-
+	f.Verbose = true
```

Comment:
I don't think we turn this on in testing as will put a lot of noise into the console.